src/commands/init.js

```diff
@@ -248,22 +259,21 @@ function interactiveMenu(directory, list) {
     });
 }
 
-function checkFolder(force = false, directoryName = '') {
+function checkFolder(force = false, directory = '') {
     return new Promise((resolve) => {
-        const directoryPath = path.join(process.cwd(), directoryName);
+        const directoryPath = getAbsolutePath(directory || process.cwd());
```

Comment:
I'm kinda on it in my __next__ branch where I make the needed changes for brining us to a stable 1.0. And I think we will need to wait with this change for that since I now have looked more closely into this and there is extensions that expects to get undefined in some places. But feel free to create a issue to make sure this is tracked correctly! 

I will go ahead and merge this if the build passes! 👍